phi/eval/eval.py

```diff
@@ -208,12 +215,59 @@ def print_result(self, answer: Optional[Union[str, Callable]] = None) -> Optiona
             title_style="bold sky_blue1",
             title_justify="center",
         )
-        table.add_row("Question", self.question)
-        table.add_row("Answer", self.answer)
-        table.add_row("Expected Answer", self.expected_answer)
+        table.add_row("Question", question)
+        table.add_row("Answer", answer)
+        table.add_row("Expected Answer", expected_answer)
         table.add_row("Accuracy Score", f"{str(result.accuracy_score)}/10")
         table.add_row("Accuracy Reason", result.accuracy_reason)
+        table.add_row("Relevance Score", f"{str(result.relevance_score)}/10")
+        table.add_row("Completeness Score", f"{str(result.completeness_score)}/10")
+        table.add_row("Confidence Score", f"{str(result.confidence_score)}/1.0")
         table.add_row("Time Taken", f"{response_timer.elapsed:.1f}s")
         console.print(table)
 
         return result
+    
+    def eval_tool_calls(self, expected_tool_calls: List[str], agent: Optional[Agent] = None, prompt: Optional[str] = None, response: Optional[RunResponse] = None) -> bool:
+        from phi.cli.console import console
+        from rich.table import Table
+        from rich.box import ROUNDED
+
+        if agent:
+            if prompt is None:
+                raise ValueError("Prompt is required to evaluate tool calls")
+            else:
+                response = agent.run(prompt)
+        elif response is None:
+            raise ValueError("Response is required to evaluate tool calls")
+        
+        last_tool_calls = None
+        for message in reversed(response.messages):
+            if message.tool_calls:
+                if last_tool_calls is None:
+                    last_tool_calls = message.tool_calls
+                else:
+                    last_tool_calls.append(message.tool_calls)
+
+        table = Table(
+            box=ROUNDED,
+            border_style="blue",
+            show_header=False,
+            title="[Tool Calls Eval Result]",
+            title_style="bold sky_blue1",
+            title_justify="center",
+        )
+        # table.add_column("Tool Call")
+        # table.add_column("Result")
+
+        failed = False
+        for tool_call in last_tool_calls:
```

Comment:
Possibly extension (for future) would be to eval the actual parameters that were passed.